docs/react-wiki-archive/BestPractices/Testing-with-Jest.md

```diff
@@ -0,0 +1,234 @@
+> 🚨🚨 **This page is primarily about `@fluentui/react-components` ("v9") and `@fluentui/react` ("v8") and related packages.** 🚨🚨&nbsp; [See this page for `@fluentui/react-northstar` ("v0").](https://github.com/microsoft/fluentui/blob/master/packages/fluentui/test-a-feature.md)
```

Comment:
this should be re-worked/moved to v9 docs

Reply:
PR Buddy == Martin

Reply:
PR: https://github.com/microsoft/fluentui/pull/31554

Reply:
completed!